lib/launchers/base.js

```diff
@@ -41,7 +40,7 @@ function BaseLauncher (id, emitter) {
 
     this.error = null
     this.state = BEING_CAPTURED
-    this.emit('start', url + '?id=' + this.id + (helper.isDefined(self.displayName) ? '&displayName=' + encodeURIComponent(self.displayName) : ''))
+    this.emit('start', url + '?id=' + this.id + (helper.isDefined(this.displayName) ? '&displayName=' + encodeURIComponent(this.displayName) : ''))
   }
 
   this.kill = function () {
```

Comment:
I think you should also change this.start and this.kill to be arrow functions to be on the safe side.